doc/man3/EVP_DigestInit.pod

```diff
@@ -88,12 +90,25 @@ Cleans up digest context B<ctx> and frees up the space allocated to it.
 
 =item EVP_MD_CTX_ctrl()
 
+This is a deprecated option. EVP_MD_CTX_get_params() and EVP_MD_CTX_get_params()
```

Comment:
`EVP_MD_CTX_get_params()` is mentioned twice.  One should be `EVP_MD_CTX_set_params()`.